conf/application.conf

```diff
@@ -197,6 +203,10 @@ object-store.default-retention-period = "1-week"
 
 internal-auth.token = "123457"
 
+internal-auth {
+    enabled = false
```

Comment:
you could combine the config above with this one like this:
```
internal-auth {
  // used for talking to object-sotre
  token = ... 
  // used for matching API
  enabled = false 
}
```
with maybe the comments to clarify what they're for

Reply:
yes , i can do that

Reply:
[updated](https://github.com/hmrc/hec/pull/55/commits/3dd88d65dfd4c7df63b233da1cc9608d0ee25a1d)